www/nginx/Makefile

```diff
@@ -1,6 +1,6 @@
 PLUGIN_NAME=		nginx
 PLUGIN_VERSION=		1.29
-PLUGIN_REVISION=	1
+PLUGIN_REVISION=	2
```

Comment:
if you want to add a feature, change the upper version to 1.30 instead and delete this line as this is for hotfixes